.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,52 @@
+name: Bug report
+description: File a bug report
+title: '[Bug]: '
+labels: [bug]
+assignees: ''
+body:
+  - type: input
+    attributes:
+      label: SDK Version
```

Comment:
Could you make this form less focused on SDK?
This repository is a monorepo containing SDKs (plural), documentation, and more to come.

Also, the goal of this form is to allow users to report any Devopness bugs, in any platform module, and not to report only SDK issues.

Reply:
I've updated the description to explain that the user should fill the SDK version only if the bug being reported is found in the SDK. Ideally, there would be a radio button to choose whether the bug is SDK related or not and if it is, the SDK Version input would open up. But GitHub issue forms do not support such form behavior.

Another possibility would be two bug report forms, one for SDK related bugs, another one for anything else.

Reply:
> Ideally, there would be a radio button to choose whether the bug is SDK related or not and if it is, the SDK Version input would open up. 
> But GitHub issue forms do not support such form behavior.

Wouldn't that be feasible, the same way done here [1] for question "Which @angular/* package(s) are the source of the bug?"

[1] https://github.com/angular/angular/issues/new?assignees=&labels=&projects=&template=1-bug-report.yaml

Reply:
But this does something different from what I had in mind. Anything you might select from that dropdown is not going to dictate if an input will show up or not. 

The problem we have is that when the bug being reported is on the SDK, we have to know which version. But if the bug has nothing to do with the SDK, we don't need to know the SDK version.

Reply:
Got you! Yeah, that would be nice.
For now perhaps having a dropdown with options `Devopness API`, `Devopness website`, `Devopness web app`, `Documentation`, `SDK-JS` could be a good starting point .
Could you apply that?

Reply:
New dropdown field added with the suggested options plus a "Don't know / other" field.